module-3/web-app/src/components/Pages/Projects/Projects.jsx

```diff
@@ -18,8 +18,8 @@ const Projects = () => {
             {projects.length > 0 && projects.map((project, i) => (
                 <Link key={i} to={`/documents/?projectid=${project.id}`}>    
                     <div className={style.project}>
-                        <div className={style.title}>{project.title}</div> 
-                        <div>{project.description}</div> 
+                        <h1 className={style.title}>{project.title}</h1> 
```

Comment:
It is inappropriate to use `h1` tag here, `h` suppose to be the main, most significant title on the page. File name underneath the file icon is definitely not the one. Such code would be incompatible with screen reader devices. 

https://en.ryte.com/product-insights/how-to-use-html-heading-tags-correctly

https://medium.com/swlh/html-heading-tags-best-practices-78a85f63df47